packages/lib-classifier/src/store/SubjectStore.spec.js

```diff
@@ -64,84 +67,82 @@ describe('Model > SubjectStore', function () {
       })
 
       it('should request more subjects', function () {
-        expect(populateSpy).to.have.been.calledOnce()
+        // Once for initialization and once after the queue has been advanced to less than 3 subjects
+        expect(rootStore.subjects.populateQueue).to.have.been.calledTwice()
+      })
+
+      describe('when the initial response has less than three subjects', function () {
+        let rootStore
+        before(function () {
+          const clientStub = stubPanoptesJs({ workflows: workflow, subjects: shortListSubjects })
+          rootStore = setupStores(clientStub)
+        })
+
+        it('should request more subjects', function () {
+          // Once for initialization and again since less than three subjects in initial response
+          expect(rootStore.subjects.populateQueue).to.have.been.calledTwice()
```

Comment:
This is passing because `populateQueue` was called twice in an earlier test.

Reply:
I think I was confused by the nested variables here. This is a fresh instance of `rootStore`, with a new spy?

Reply:
Each describe block has its own root store instantiated.